### DISCUSSION TITLE
DietPi-Software | SteamCMD

### DISCUSSION THREAD

**Discussion by CalibenQC** [MichaIng/DietPi#6062] [Software Request]
I have a suggestion of software :SteamCMD. it can be use for create some servers for some steam games

**Comment by MichaIng** [comment]
Many thanks for your suggestion. Do you have some project links, for source code, install instructions or so?

**Reply by nonnorm** [reply]
Just an APT package (https://packages.debian.org/bullseye/steamcmd). Similar to Steam itself, `i386` only, so would need to be run with `box86` on ARM.

**Reply by MichaIng** [reply]
Ah okay. Requires an `armhf` DEB package as well then, to host on our servers, like we do for `steam`. That one actually needs an update.
